app/server/api/organisation_api.py

```diff
@@ -76,6 +75,7 @@ def post(self, organisation_id):
 
         new_organisation = Organisation(name=name, token=token)
 
+        # TODO: Shift this all into being created when an org is created
```

Comment:
yeah i actually put it in there, but then moved it back out again because from memory there were some weird side effects i ran across when originally writing this section, and i didn't wanna change it without testing properly